src/Compilers/Core/Portable/Syntax/GreenNode.cs

```diff
@@ -39,18 +39,7 @@ private string GetDebuggerDisplay()
         private NodeFlagsAndSlotCount _nodeFlagsAndSlotCount;
         private int _fullWidth;
 
-        protected NodeFlags flags
-        {
-            get => _nodeFlagsAndSlotCount.NodeFlags;
-            set => _nodeFlagsAndSlotCount.NodeFlags = value;
-        }
-
-        /// <inheritdoc cref="NodeFlagsAndSlotCount.SmallSlotCount"/>>
-        private byte _slotCount
-        {
-            get => _nodeFlagsAndSlotCount.SmallSlotCount;
-            set => _nodeFlagsAndSlotCount.SmallSlotCount = value;
-        }
+        internal NodeFlags Flags => _nodeFlagsAndSlotCount.NodeFlags;
```

Comment:
I'd like to make this private.  But VB has a couple of usages realted to caching tokens where it directly introspects this flag.